source/_components/switch.lupusec.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: "Lupusec Switch"
+description: "Instructions on how to integrate Lupusec switches into Home Assistant."
+date: 2018-10-03 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lupusec.jpg
+ha_release: 0.79
```

Comment:
`0.79` has been shipped already some time ago. The next possible release would be `0.81`.